lib/lotus/repository.rb

```diff
@@ -379,7 +380,7 @@ def update(entity)
       #
       #   ArticleRepository.delete(article) # raises Lotus::Model::NonPersistedEntityError
       def delete(entity)
-        if entity.id
+        if entity.persisted?
```

Comment:
after talking to @jodosha, it is decide to remove `Entity#persisted?` method because Entity should not be aware of any persistence logic. Please revert it to `entity.id` like before.